### ISSUE TITLE
Docs organization on napari.org: information is hard to find, for example Shapes layer tutorial.

### ISSUE THREAD

**Issue by psobolewskiPhD** [napari/docs#42]
## 🐛 Bug

Docs organization between user centric vs. dev centric, basic vs. advanced isn't clear. 

Just to illustrate:
I wanted to clarify something about the Shapes layer and I recalled there was a dedicated tutorial for it. I went to look in the Usage docs on napari.org, but didn't find it. 
So I tried search. This wasn't helpful:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/76622105/200166507-c945572f-609d-484f-9bbf-76bca2fc2289.png">

BTW google get's it correct on the first hit:
<img width="714" alt="image" src="https://user-images.githubusercontent.com/76622105/200165831-e0e18a1e-a613-46f6-838c-d404f62dd25f.png">

I then went to Getting started > Layers at a glance, but the only clickable links I found there re: shapes are to the API docs.

Eventually I found what I was looking for by just opening all the submenus:
<img width="312" alt="image" src="https://user-images.githubusercontent.com/76622105/200165568-9756a668-6bca-44b7-9830-0e807226cc46.png">

Turns out the Shapes tutorial is under How-to Guides. OK. But the rest of those how-to guides are considerably more advanced and (a bit esoteric?) for a totally different audience than the layer tutorials:
<img width="229" alt="image" src="https://user-images.githubusercontent.com/76622105/200165989-56c87dac-785c-4d50-8771-1a4d1397bc6f.png">

For the record, the Tutorials section is well organized and these are aimed at intermediate to advanced **users**.

Finally, the section "In Depth explanations" is indeed in depth, but these are really advanced topics, rather not user facing—or rather targeting an advanced user looking to extend napari.

TLDR: the organization and naming of: Getting started, Tutorials, How-to guides, In depth Explanations isn't clear. It's not clear what the differences in content should be in each of those menus nor what audience (user level) is targeted by which menu—other than the first.

## To Reproduce

Browse the Usage menu of napari.org

## Expected behavior

A napari user of every level: new, beginning, intermediate, advanced should be able to find appropriate documentation. Additionally, the search should be more functional.

## Environment

N/A

## Additional context

Layers are—for me—the core thing of napari, making it different from say ImageJ/Fiji.
I would strongly consider breaking out Using Layers into it's own top level menu below getting started. The remainder of the How-to guides could maybe be squeezed into Tutorials or In-depth.

**Comment by brisvag** [comment]
I agree on the whole line; I often find myself just googling or using firefox's history instead. We could chat about it in the next community meeting, or do you already have a proposal for how to reorganize?

**Comment by psobolewskiPhD** [comment]
Nothing beyond what I put in `Additional Context` about moving the Using Layers to it's own level.

**Comment by melissawm** [comment]
@psobolewskiPhD I think it would be really interesting to have this discussion at the Docs working group meeting at some point. Feel free to bring it up at the community meeting as well. My initial organization here was based on document type: tutorials vs. how-to etc. But having a second layer of organization into basic/advanced may be needed.

My thinking is that without modifying content, these layers guides are actually how-tos, but maybe we *also* want to create tutorials for them? Since they are so fundamental to the napari usage, this may be a good idea. What do you think?

Pinging also @jni and @neuromusic from the docs wg.

**Comment by psobolewskiPhD** [comment]
@melissawm I guess I don't quite get the differences between a how-to and a tutorial. I've read and or used most of them and honestly I can't say I see a major difference in style. 
For what use case should a user be looking for one vs the other? 

Regarding the layer guides: they're pretty perfect I think, I'm just advocating we surface them into a top level position under `Getting Started` because they are the key, core feature of napari. This makes it easier for a (new) user to go from `Getting started > Layers at a glance` to `Using Layers > Using the Shapes layer` (for example).

(BTW, should `napari workshops` maybe get pushed down the menu? It's a bit odd there between tutorial and how-to)

**Comment by melissawm** [comment]
I've been thinking of ways to solve this and had an idea - maybe not the best but I wanted to propose it anyway. 

How about 
- pushing napari-workshops a bit down the menu
- keeping the layers how-tos where they are
- highlighting both the napari-workshops and layers pages through *cards* in the Usage main page:

<img width="1373" alt="Screenshot 2022-11-29 at 8 40 13 PM" src="https://user-images.githubusercontent.com/3949932/204672394-c0eb82b5-20cf-4de6-86b4-2c624093144a.png">

These cards are providade by [sphinx-design](https://sphinx-design.readthedocs.io/en/latest/cards.html), which supersedes both `sphinx-tabs` and `sphinx-panels` that we already use, so this is a change we need to make at some point anyway. Let me know if this would be reasonable.

**Comment by melissawm** [comment]
To answer your question about tutorials and how-tos: the whole idea is based on [the Diataxis framework](https://diataxis.fr/), and the idea is:
- Tutorials are learning-oriented and usually aimed at beginners or people who want to learn concepts and basic techniques
- How-to guides are task-oriented and are usually shorter and more to-the-point, listing how to achieve certain tasks

My reading is that the layer guides are a mix of both, and they are hard to classify. They involve basic topics, but read like how-tos (here's how to add an image; here's how to load multiscale images; here's how to load grayscale images etc)

**Comment by psobolewskiPhD** [comment]
@melissawm I love the cards! 😍 
I think surfacing the layers guides that way looks great to me.
Thanks for the info on the rationale behind it. I was more looking at the content and the other how-to's are *really advanced* stuff that I don't really get—certainly not for a novice—as compared to the layers ones which are really excellent and targeted a newer user.
So I guess I still feel like the layers guides deserve their own top-level position, but the card does that, so good to go.

**Comment by melissawm** [comment]
I am closing this issue as completed since #63 was merged, but feel free to reopen if there are any leftover items to resolve.
